setup.py

```diff
@@ -2,19 +2,30 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-
 from setuptools import setup
 
-
-setup(
-    name='FoxPuppet',
-    version='0.1.0',
-    description='Firefox user interface testing model for use with Selenium',
-    long_description=open('README.rst').read(),
-    url='https://github.com/mozilla/FoxPuppet',
-    license='MPL2',
-    packages=['foxpuppet'],
-    install_requires=[
-        'selenium>=3.0.1'
-    ]
-)
+setup(name='FoxPuppet',
+      use_scm_version=True,
+      description='Firefox user interface testing model for use with Selenium',
+      long_description=open('README.rst').read(),
+      author='Firefox Test Engineering',
+      author_email='firefox-test-engineering@mozilla.com',
+      url='https://github.com/mozilla/FoxPuppet',
+      packages=['foxpuppet'],
+      install_requires=['selenium>=3.0.1'],
```

Comment:
I haven't introduced that requirement in this patch. I feel the dependency should only be bumped alongside the patch that uses the new context manager.